fix the additional hardcoded compiler options that are making trouble.
author: fenglai liu (fenglai@accre.vanderbilt.edu)
diff -Naur augustus-3.3.2.ori/auxprogs/bam2hints/Makefile augustus-3.3.2/auxprogs/bam2hints/Makefile
--- augustus-3.3.2.ori/auxprogs/bam2hints/Makefile	2019-02-26 13:31:59.005083190 -0600
+++ augustus-3.3.2/auxprogs/bam2hints/Makefile	2019-02-27 11:58:22.100807318 -0600
@@ -8,13 +8,13 @@
 #	Last modified:  09-October-2015 by Katharina J. Hoff
 
 # Variable definition
-INCLUDES = /usr/include/bamtools
+INCLUDES = $(EBROOTBAMTOOLS)/include/bamtools
 LIBS = -lbamtools -lz
 SOURCES = bam2hints.cc 
 OBJECTS = $(SOURCES:.cc=.o)
 CXXFLAGS += -Wall -O2 # -g -p -g -ggdb 
 
-LINK.cc = g++
+LINK.cc = $(CXX)
 
 # Recipe(s)
 # $@: full target name of current target. 
diff -Naur augustus-3.3.2.ori/auxprogs/filterBam/src/Makefile augustus-3.3.2/auxprogs/filterBam/src/Makefile
--- augustus-3.3.2.ori/auxprogs/filterBam/src/Makefile	2019-02-26 13:31:59.004083210 -0600
+++ augustus-3.3.2/auxprogs/filterBam/src/Makefile	2019-02-27 12:23:43.812006140 -0600
@@ -8,7 +8,7 @@
 			printElapsedTime.cc sumMandIOperations.cc sumDandIOperations.cc PairednessCoverage.cc
 PROGRAM = filterBam
 OBJECTS = $(SOURCES:.cc=.o)
-BAMTOOLS = /usr/include/bamtools
+BAMTOOLS = $(EBROOTBAMTOOLS)/include/bamtools
 INCLUDES = -I$(BAMTOOLS) -Iheaders -I./bamtools
 LIBS = -lbamtools -lz
 CFLAGS = -std=c++0x
